### PR TITLE
Readme cleanup

### PR DESCRIPTION
**Pull request by hanssto**
* More idiomatic English.
* Remove unnecessary capitalization.
* Make lists of params into actual lists for readability.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,8 +95,13 @@ on which there are less than 3 prices in total.
...
 Implement an API endpoint where you can upload a price, including
-the following parameters: *date_from, date_to, origin_code,
-destination_code, price*
+the following parameters:
+
+* date_from
+* date_to
+* origin_code,
+* destination_code
+* price*
```
Comment (ador) [inline README.md]: There's a star character after "price", not needed anymore.

File: README.md [hunk 2]
```diff
@@ -122,21 +127,20 @@ you need to take into consideration?
...
   complicated date processing.
 
-* You are encouraged to modify or extend the database schema if you think a different model fits task better.
+* You are encouraged to modify or extend the database schema if you think a different model fits the task better.
 
 * If you have any questions, please don't hesitate to contact us at tech-recruitment@xeneta.com
 
-* We would like your feedback - Let us know how much time you spent on
-  the task or about any difficulties you run into.
+* We would like your feedback - Let us know how much time you spent on the task, or any difficulties you ran into.
```
Comment (prmrdl) [inline README.md]: I really don't like this — it feels unfair. We're asking them to improve our process after having them do unpaid work in hopes of getting a job? I'd suggest removing it.

File: README.md [hunk 3]
```diff
@@ -7,46 +7,50 @@ database dump is provided that includes the following information:
...
+* 5-character destination port code
+* The day for which the price is valid
 * The price in USD
 
 # Assignment
 
 Develop an HTTP-based API capable of executing the tasks described
 below. Our stack is based on Flask, but you are free to choose
-anything you like. All data returned is expected to be in JSON format.
+anything you like. All data returned are expected to be in JSON format.
```
Comment (prmrdl) [inline README.md]: So while technically the correct grammar is "data are returned", you will find that in almost any software publication the word "data" is treated as a singular. As such, it really should be written as "All data returned is expected to be..."

File: README.md [hunk 4]
```diff
@@ -105,11 +114,7 @@ a free API for retrieving currency exchange information.
 
 ## Batch processing
```
Comment (prmrdl) [inline README.md]: From what I understand, this is a third standalone task that is unrelated to the GET and POST parts, right? 

As it stands now, it's not very clear to me that this is the case. Perhaps structure the other parts with numbers:

1) GET Requests
2) POST Requests
3) Batch Processing

File: README.md [hunk 5]
```diff
@@ -7,46 +7,50 @@ database dump is provided that includes the following information:
...
-* 5-character Destination Port Code
-* The day on which the price is valid on
+* 5-character origin port code
+* 5-character destination port code
+* The day for which the price is valid
 * The price in USD
 
 # Assignment
 
 Develop an HTTP-based API capable of executing the tasks described
```
Comment (prmrdl) [inline README.md]: Suggestion for clarity: "Develop an HTTP-based API capable of executing the GET and POST tasks described..."

File: README.md (hunk 2 above)
Comment (zeveshe) [reply README.md:137]: I agree this is unfair, but this text is just wrong: this is not about feedback.
We want to know how long it took to finish the task and what the difficulties were because this is useful input for the technical interview after the task is submitted.

**Review by prmrdl — APPROVED** [review]
LGTM!
